src/main/cpp/events/baseaudioevent.h

```diff
@@ -153,20 +159,22 @@ class BaseAudioEvent
 
         // for backwards compatibility we provide these methods
         // use setEventLength(), setEventStart()... getEventEnd() instead.
-        // deprecation keyword is sadly C++14 extension
+        // deprecation keyword is a C++14 extension and sadly not supported by SWIG...
 
-        //[[deprecated("use setEventLength instead")]]
+#ifndef SWIG
+        [[deprecated("use setEventLength instead")]]
         void setSampleLength( int value );
-        //[[deprecated("use setEventStart instead")]]
+        [[deprecated("use setEventStart instead")]]
         void setSampleStart( int value );
-        //[[deprecated("use setEventEnd instead")]]
+        [[deprecated("use setEventEnd instead")]]
         void setSampleEnd( int value );
-        //[[deprecated("use getEventLength instead")]]
+        [[deprecated("use getEventLength instead")]]
         int getSampleLength();
-        //[[deprecated("use getEventStart instead")]]
+        [[deprecated("use getEventStart instead")]]
         int getSampleStart();
-        //[[deprecated("use getEventEnd instead")]]
+        [[deprecated("use getEventEnd instead")]]
         int getSampleEnd();
+#endif
```

Comment:
About to deprecate these legacy function names (have now been removed from SWIG wrapping, thus unavailable through JNI)